op-monitorism/faultproof_withdrawals/runbooks/RUNBOOK.md

```diff
@@ -0,0 +1,217 @@
+
+- **Description:** This is the total number of suspicious withdrawals. In normal circumstances, this metric will be incremented by 1, and `faultproof_withdrawals_potential_attack_on_in_progress_games_count` will be decremented by 1.
+- **Type:** Counter
+- **Alert:**
+  - **Condition:** If the value is not increasing as expected.
+  - **Action:** No immediate actions are required. This information is useful for threat hunting and measuring how many attack attempts have been detected since monitoring began.
+  - **Alert Name:** [faultproof-suspicious-withdrawal-forgery-detected](#faultproof-withdrawal-forgery-detection-stalled)
+
+### `faultproof_withdrawals_node_connection_failures_total`
+
+- **Description:** Total number of node connection failures.
+- **Alert:**
+  - **Condition:** If the value increases over time.
+  - **Action:** Investigate network issues or node outages. Check logs for connection errors and attempt to reconnect.
+  - **Alert Name:** [faultproof-withdrawal-forgery-detection-error-unhandled](#faultproof-withdrawal-forgery-detection-error-unhandled)
+
+### `faultproof_withdrawals_events_processed_total`
+
+- **Description:** Number of withdrawal events processed. If the number of withdrawals observed on the chain drops below a certain threshold (e.g., usually more than 1 per day), this alert will be triggered.
+- **Alert:**
+  - **Condition:** If the metric's increase falls below a set threshold.
+  - **Action:** Investigate whether the chain had withdrawals during the relevant period to confirm whether the issue is related to monitoring or a lack of activity on the network.
+  - **Alert Name:** [faultproof-withdrawal-forgery-detection-stalled](#faultproof-withdrawal-forgery-detection-stalled)
+
+### `faultproof_withdrawals_withdrawals_processed_total`
+
+- **Description:** Number of withdrawals processed. These withdrawals are complete, and they are forgotten.
+
+### `faultproof_withdrawals_initial_l1_height`
+
+- **Description:** Indicates the initial L1 (Layer 1) block height at the start of the monitoring period.
+
+### `faultproof_withdrawals_invalid_proposal_withdrawals_events_count`
+
+- **Description:** Tracks the number of invalid proposal withdrawal events.
+
+### `faultproof_withdrawals_latest_l1_height`
+
+- **Description:** Indicates the latest observed L1 block height.
+
+### `faultproof_withdrawals_next_l1_height`
+
+- **Description:** Represents the next expected L1 block height.
+
+---
+
+## Incident Response
+
+- **Investigate System Logs:** For each alert, review the logs from the affected services to gather more information.
+- **Check Node Health:** Verify the status of all nodes involved, especially if connection failures are reported.
+- **Coordinate with Relevant Teams:** Communicate with development, operations, and security teams as necessary.
+- **Document Findings:** Record the incident details, root cause, and resolution steps for future reference.
+
+---
+
+## Conclusion
```

Comment:
The "Incident Response" and "Conclusion" sections feel out of place. Should "Incident Response" be moved towards the top when explaining how to use the runbook? And "Conclusion" feels like it can be deleted